common/luaclass.h

```diff
@@ -123,7 +123,7 @@ luaA_checkudataornil(lua_State *L, int udx, lua_class_t *class)
     static inline int                                                          \
     luaA_##prefix##_class_add_signal(lua_State *L)                             \
     {                                                                          \
-        signal_add(&(lua_class).signals, luaL_checkstring(L, 1));              \
+        luaA_deprecate(L, "signal usage without add_signal()");                \
```

Comment:
The problem is the same one that we have with `awful.util.deprecate`: The implementation does "This function is deprecated and will be removed, see <the argument here>". That's why I'm using this weird wording.

Should I instead change `luaA_deprecate`?